book.js

```diff
@@ -0,0 +1,64 @@
+class BookList {
+  constructor(
+    bookListElement,
+    addBookFormElement,
+    titleInputElement,
+    authorInput,
+  ) {
+    this.bookListElement = bookListElement;
+    this.addBookFormElement = addBookFormElement;
+    this.titleInputElement = titleInputElement;
+    this.authorInput = authorInput;
+    this.books = [];
+    this.loadBooks();
+    this.updateBookList();
+    this.addBookFormElement.addEventListener('submit', this.add.bind(this));
+  }
+
+  loadBooks() {
+    const storedBooks = JSON.parse(localStorage.getItem('books'));
+    if (storedBooks) {
+      this.books = storedBooks.map((book) => ({
+        title: book.title,
+        author: book.author,
+      }));
+    }
+  }
+
+  add(event) {
+    event.preventDefault();
```

Comment:
- [ ] According to the requirements you should create a directory called modules and put each separate module you created in this directory.